packages/paste-website/src/layouts/DefaultLayout.tsx

```diff
@@ -58,8 +58,10 @@ const DefaultLayout: React.FC<DefaultLayoutProps> = ({children, pageContext, loc
       </Box>
       <Box
         paddingTop={['space40', 'space40', 'space100']}
-        paddingX={['space50', 'space70', 'space200']}
+        paddingX={['space70', 'space200', 'space200']}
         backgroundColor="colorBackgroundBody"
+        maxWidth="73.5rem" // custom spacing value: size100(64rem), + (space200(4.75rem) * 2)
```

Comment:
quick maths

Reply:
What's this used for? We tried to use sizing tokens for everything but maybe we missed something?

Reply:
@katieporter padding will take away from an element's available width. In the comps, 1024px was declared for the width of the content, but we also have padding on that element, so we have to add it to the total width for the element.

The rendering sizes for this ends up like this:

![Screen Shot 2021-11-18 at 1 39 13 PM](https://user-images.githubusercontent.com/36438/142485480-a84f1188-a821-40a6-a6e2-7b64a1120f0d.png)

Reply:
Ahhh gotcha, thanks for explaining. I didn't realize we were using padding. Can we use margins instead so we can stick to using token values, or is there a specific reason we chose to pad it?

Reply:
We use auto margins to center the element, so we're sort of restricted there.

We could change the box model sizing for the element. This might actually be an appropriate use for the property. 🤔